### PR TITLE
#168211855 Create the Apps homepage

### PR DESCRIPTION
**Pull request by cvjude**
#### What does this PR do?
- Create the App home page

#### How can this be manually tested?
- Clone the repo
- Run the command below in your terminal to install all dependencies
```
npm install
``` 
- Run the command below to test
```
npm test
```

#### Any background context you want to provide?
none

#### What are the relevant Pivotal Tracker stories? 
- [#168211855](https://www.pivotaltracker.com/story/show/168211855)

### CODE REVIEW

**Review by obayomi96 — APPROVED** [review]
This is very cool @cvjude,  great work LGTM

File: src/actions/Articles/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    const response = await axois.get(
+      `${APP_URL}/api/v1/articles/categories/article`,
+    );
+    dispatch({
+      type: 'GET_HOME_ARTICLES_FROM_DATABASE',
+      payload: response.data,
+    });
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.log(err);
```
Comment (chiboycalix) [inline src/actions/Articles/index.js]: I think you forgot to remove this console.log statement.

**Review by Hadeneekeh — APPROVED** [review]
LGTM!!!

**Review by chiboycalix — APPROVED** [review]
LGTM

**Review by encodedBicoding — APPROVED** [review]
LGTM

**Review by Lundii — APPROVED** [review]
LGTM

**Pull request merged**